client/app/components/permissions-editor/PermissionsEditorDialog.jsx

```diff
@@ -77,7 +77,7 @@ function UserSelect({ onSelect, shouldShowUser }) {
   useEffect(() => {
     setLoadingUsers(true);
     debouncedSearchUsers(searchTerm);
-  }, [searchTerm]);
+  }, [debouncedSearchUsers, searchTerm]);
```

Comment:
Probably it should be replaced with `use-debounced` :thinking: But let's keep as is for now